lib/identity_doc_auth/acuant/requests/create_document_request.rb

```diff
@@ -28,7 +30,7 @@ def body
               },
             },
             ImageCroppingExpectedSize: '1',
-            ImageCroppingMode: '1',
+            ImageCroppingMode: cropping_mode,
```

Comment:
do we need to `.to_s` here to match the type? or make the constants into strings?

Reply:
> do we need to `.to_s` here to match the type? or make the constants into strings?

I'd _hoped_ it wouldn't make a difference, given how some of the other "mode" configurations are set as numbers, and how they're defined as numbers in the SDK documentation for the `CroppingMode` enum (AssureID Connect SDK Integration Guide, p. 20). But just to be safe, I'll restore it to assign as string.

Reply:
> do we need to `.to_s` here to match the type? or make the constants into strings?

Updated in ac9ef18.